crates/proto/protobuf/ya_relay.proto

```diff
@@ -168,6 +170,7 @@ message Response {
         repeated Endpoint endpoints = 4;
         uint32 seen_ts = 5;
         uint32 slot = 6;
+        repeated bytes supported_encryptions = 7;
```

Comment:
There is a problem. In case of forwarding through relay, how will Node decide which algorithm to use and how the second Node will know which was used??
I wrote in spec that we should have only one value for now, so there is no ambiguity.

Reply:
after a call we decided that both sides share the list, chosen_encryption needs to be calculated based on the lists or added as a new field.